### ISSUE TITLE
enunciate 2.13.0 generates incorrect swagger.json

### ISSUE THREAD

**Issue by mnisius** [stoicflame/enunciate#1027]
Hello, I've noticed that the new version generates an incorrect swagger.json

This is my rest controller. 

```java
@RestController
@RequestMapping(value = "rest")
public class ServiceApp {

  @GetMapping(value = "/getTest")
  public String hellWorld() {
    return "Hello World";
  }

  @PostMapping(value = "/postTest")
  public String hellWorld(@RequestBody String test) {
    return test; 
  }
}
```

So I have 1 GET service and 1 POST service . But in the swagger.json every service  is registered in each http method. So my swagger claims that there are 4 endpoints. 2 GET Services and 2 POST Services. So this is how my swagger page looks

![proof](https://user-images.githubusercontent.com/56688244/82451366-3147d680-9aae-11ea-8c06-acd6b3c56bfc.png)

With enunciate 2.12.1 the swagger.json is correct

**Comment by stoicflame** [comment]
Yes, this has already been noted and fixed at 00cd32ab61ff74e5a4f73fa318aa411f0d7672d5.

I'll do a patch release in the next week or so.

**Comment by stoicflame** [comment]
Fix is available in 2.13.1.
